src/Avalonia.Base/Input/GestureRecognizers/GestureRecognizer.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Avalonia.Input.GestureRecognizers
+{
+    public abstract class GestureRecognizer : StyledElement
+    {
+        public abstract IInputElement? Target { get; }
```

Comment:
Both `Target` and `Initialize` should be handled by the base class, IMO. 

e. g. 
```
protected internal IInputElement? Target { get; internal set; }
```
with `Initialize` being removed completely.